src/ngraph_rewrite_pass.cc

```diff
@@ -150,6 +151,16 @@ class NGraphVariableCapturePass : public NGraphRewritePass {
       DumpGraphs(options, idx, "captured", "Graph With Variables Captured");
     }
 
+    // Skip "Assert" if specifically asked by the user
+    if (std::getenv("NGRAPH_TF_SKIP_ASSERT") != nullptr) {
```

Comment:
Need to change this to `NGRAPH_TF_DISABLE_ASSERTS`. Also add a VLOG(0) indicating that the model is running with asserts disabled.